.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: java
+cache:
+  directories:
+  - $HOME/.m2
+jdk:
+  - openjdk8
```

Comment:
This is a controlled image that we 100% control. Why bother building on several JDKs?